### PR TITLE
Handling SIGINT error to not to exit abnormally

### PR DESCRIPTION
**Pull request by lonewolf3739**
Handling  SIGINT errors.

### CODE REVIEW

File: Main.py [hunk 1]
```diff
@@ -9,7 +9,12 @@
 from ExtractMainContent import ExtractMainContent
 import sys
 import codecs
+import signal
 
+def signal_handler(signal, frame):
```
Comment (Griffintaur) [inline Main.py]: can you add a comment for this and its purpose?

It is good to maintain a uniform indentation.

File: NewsPulling.py [hunk 2]
```diff
@@ -30,6 +30,10 @@ def PullNews(self):
             print "A connection Attempt failed"
             print e.message
             sys.exit()
+      	except TimeoutError as e:
```
Comment (Griffintaur) [inline NewsPulling.py]: alignment should be uniform.

File: Main.py (hunk 1 above)
Comment (srikanthchekuri) [inline Main.py]: yes i'll add comment and follow uniform indentation

File: NewsPulling.py (hunk 2 above)
Comment (srikanthchekuri) [inline NewsPulling.py]: it actually looked uniform but it looks different here

File: Main.py [hunk 3]
```diff
@@ -9,7 +9,16 @@
 from ExtractMainContent import ExtractMainContent
 import sys
 import codecs
-
+import signal
+"""
+this function handles signal SIGINT generated when user presses ctrl-c
```
Comment (Griffintaur) [inline Main.py]: In Python, there is a general guideline that doc string of the function should be inside the function definition part.
for e.g  def signal ():
               """this function does these things"""

File: Main.py [hunk 4]
```diff
@@ -9,7 +9,16 @@
...
+import signal
+"""
+this function handles signal SIGINT generated when user presses ctrl-c
+if SIGINT is not handled manually abnormal termination occurs
+"""
+def signal_handler(signal, frame):
+    print('\nExiting gracefully')
+    sys.exit(0)
+#registering the signal SIGINT with signal_handler function    
+signal.signal(signal.SIGINT, signal_handler)
```
Comment (Griffintaur) [inline Main.py]: can you register the signal in App function instead of doing it at the global level.
Don't forget to place the comment there?

**Pull request closed without merging**